src/plugin/google/places/resultcard.js

```diff
@@ -1,47 +1,55 @@
-goog.provide('plugin.google.places.ResultCardCtrl');
-goog.provide('plugin.google.places.resultCardDirective');
+goog.module('plugin.google.places.ResultCardUI');
 
-goog.require('goog.async.Delay');
-goog.require('ol.extent');
-goog.require('os');
-goog.require('os.ui.Module');
-goog.require('os.ui.search.FeatureResultCardCtrl');
+const {ROOT} = goog.require('os');
+const Module = goog.require('os.ui.Module');
+const FeatureResultCardCtrl = goog.require('os.ui.search.FeatureResultCardCtrl');
 
 
 /**
  * The geonames result card directive for displaying search results.
  *
  * @return {angular.Directive}
  */
-plugin.google.places.resultCardDirective = function() {
-  return {
-    replace: true,
-    restrict: 'E',
-    templateUrl: os.ROOT + 'views/plugin/google/places/resultcard.html',
-    controller: plugin.google.places.ResultCardCtrl,
-    controllerAs: 'resultCtrl'
-  };
-};
+const directive = () => ({
+  replace: true,
+  restrict: 'E',
+  templateUrl: ROOT + 'views/plugin/google/places/resultcard.html',
+  controller: Controller,
+  controllerAs: 'resultCtrl'
+});
+
+/**
+ * The element tag for the directive.
+ * @type {string}
+ */
+const directiveTag = 'googleplacesresultcard';
 
 
 /**
  * Register the directive.
  */
-os.ui.Module.directive('googleplacesresultcard', [plugin.google.places.resultCardDirective]);
+Module.directive('googleplacesresultcard', [directive]);
```

Comment:
Nit: this should use the `directiveTag` instead of the raw string.

Reply:
The script doesn't use `directiveTag` here because that property is the element tag for the directive, which will not be the same as the `Module.directive` value if camel case is used. For example, a directive called `myUi` would have a directive tag of `my-ui`.